packages/ember-glimmer/tests/integration/helpers/get-test.js

```diff
@@ -0,0 +1,337 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { set } from 'ember-metal/property_set';
+import { get } from 'ember-metal/property_get';
+import EmberObject from 'ember-runtime/system/object';
+import TextField from 'ember-views/views/text_field';
+import Component from 'ember-views/components/component';
+
+moduleFor('Helpers test: {{get}}', class extends RenderingTest {
+  ['@test should be able to get an object value with a static key']() {
+    this.render(`[{{get colors 'apple'}}] [{{if true (get colors 'apple')}}]`, {
+      colors: { apple: 'red', banana: 'yellow' }
+    });
+
+    this.assertText('[red] [red]');
+
+    this.runTask(() => this.rerender());
+
+    this.assertText('[red] [red]');
+
+    this.runTask(() => set(this.context, 'colors', {
+      apple: 'green', banana: 'purple'
+    }));
+
+    this.assertText('[green] [green]');
```

Comment:
(The reset should use "replacement" like you did above: `set(this.context, 'colors', { apple: 'green' })`)